File: src/main/java/com/zzkun/util/date/MyDateFormater.java
package com.zzkun.util.date;


import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

/**
 * Created by kun on 2016/8/17.
 */
public class MyDateFormater {

    private static final DateTimeFormatter f1 = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");

    //时间转为字符串
    public static String toStr1(LocalDateTime dateTime) {
        return dateTime.format(f1);
    }

    public static LocalDateTime toDT1(String str) {
        return LocalDateTime.parse(str, f1);
    }
}

File: src/main/java/com/zzkun/util/date/MyDateFormater.kt
package com.zzkun.util.date


import java.time.LocalDateTime
import java.time.format.DateTimeFormatter


object MyDateFormater {

    private val f1 = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss")

    //时间转为字符串
    fun toStr1(dateTime: LocalDateTime): String {
        return dateTime.format(f1)
    }

    fun toDT1(str: String): LocalDateTime {
        return LocalDateTime.parse(str.trim { it <= ' ' }, f1)
    }
}
